charts/kof-storage/templates/promxy/local-promxy-server-group.yaml

```
@@ -5,10 +5,10 @@ metadata:
   labels:
     app.kubernetes.io/name: promxy-operator
     k0rdent.mirantis.com/promxy-secret-name: kof-mothership-promxy-config
-  name: promxyservergroup-sample
+  name: promxyservergroup-prometheus-storage
   namespace: kof
 spec:
-  cluster_name: storage0
+  cluster_name: prometheus-storage
```

Should match with the mothership cluster name
```suggestion
  cluster_name: mothership
```